world.js

```diff
@@ -32,6 +32,63 @@ const localMatrix = new THREE.Matrix4();
 const localMatrix2 = new THREE.Matrix4();
 const localRaycaster = new THREE.Raycaster();
 
+let pendingMonetizationStart = [];
+setInterval(() => {
+  if (pendingMonetizationStart.length > 0) {
+    pendingMonetizationStart.map((id, i) => {
+      const instanceId = pendingMonetizationStart[i].instanceId;
+      const object = pendingMonetizationStart[i].object;
+      const monetization = window.document[`monetization${instanceId}`];
+      const ethereumAddress = loginManager.getAddress();
+
+      const listener = new THREE.AudioListener();
```

Comment:
The audio part seems like it should be part of the NFT. An NFT might want totally different behavior/to play as different sound.

Reply:
That makes sense. Should there be any Webaverse UI/UX when and item is unlocked, or should we leave that to the NFT creators?

Reply:
Monetization status should be viewable in the context sidebar, but there could also be in-world popup notifications for WM specifically.

I think it should be discoverable but not intrusive into author intent.

Reply:
> Monetization status should be viewable in the context sidebar

Here's a basic version, a money sign icon on the context sidebar for items when web monetization is enabled:  df3b8f4

![monetizedicon](https://user-images.githubusercontent.com/29695350/101466013-4418e000-3906-11eb-8e7d-f79b98a5fb28.png)